src/rules/converters/no-import-side-effect.ts

```diff
@@ -0,0 +1,23 @@
+import { RuleConverter } from "../converter";
+
+export const convertNoImportSideEffect: RuleConverter = tsLintRule => {
+    const rules = [];
+    const notices = [];
+
+    if (tsLintRule.ruleArguments.length > 0) {
+        rules.push({ allow: tsLintRule.ruleArguments[1]["ignore-module"] });
```

Comment:
Oh, I just realized this is directly copying the original `ignore-module` setting over... but, as the notice mentions, they're different formats. It could be confusing to see the original regex placed in the ESLint rule options, and potentially a cause of unexpected behavior differences. IMO it'd be better to just not add an `allow` argument at all.

Sorry for not commenting on this sooner!

Reply:
No worries, I'll take a look